Tests/Route/DefaultRouteGeneratorTest.php

```diff
@@ -51,6 +51,10 @@ public function testGenerateUrl($expected, $name, array $parameters)
         $collection->add('foo');
         $collection->addCollection($childCollection);
 
+        $nochildCollection = new RouteCollection('base.Code.Child', 'admin_child', '/foo/', 'BundleName:ControllerName');
```

Comment:
Well vim + many plugins + very cool external programs I also use in cli like [ag](https://github.com/ggreer/the_silver_searcher) or [fzf](https://github.com/junegunn/fzf)